File: app/src/main/java/de/nicidienase/chaosflix/entities/streaming/Group.java
package de.nicidienase.chaosflix.entities.streaming;

import java.util.List;

/**
 * Created by felix on 23.03.17.
 */

public class Group {
	String group;
	List<Room> rooms;

	public String getGroup() {
		return group;
	}

	public void setGroup(String group) {
		this.group = group;
	}

	public List<Room> getRooms() {
		return rooms;
	}

	public void setRooms(List<Room> rooms) {
		this.rooms = rooms;
	}
}

File: common/src/main/java/de/nicidienase/chaosflix/common/mediadata/entities/streaming/Group.kt
package de.nicidienase.chaosflix.common.mediadata.entities.streaming

import androidx.annotation.Keep
import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@Keep
@JsonIgnoreProperties(ignoreUnknown = true)
data class Group(
    var group: String,
    var rooms: MutableList<Room>
) {

    companion object {
        val dummyObject: Group
            get() {
                val dummy = Group("Dummy Group", mutableListOf())
                dummy.rooms.add(Room.dummyObject)
                return dummy
            }
    }
}
